VERSIONING.md

```diff
@@ -0,0 +1,54 @@
+# Versioning Policy
+
+The AWS Encryption SDK for JavaScript versioning follows [semantic versioning][link-semver] standards.
+
+## Major versions
+
+Major version changes are significant and expected to break backwards compatibility.
+
+## Minor versions
+
+Minor version changes will not break compatibility between the previous minor versions;
+to do so is a bug.
+Encryption SDK changes will also involve addition of optional features, and non-breaking enhancements.
+Additionally any change to the version of a dependency.
```

Comment:
"Additionally, any change to the version of a dependency is a minor version change."

or

"Minor version changes additionally include any change to the version of a dependency."

I don't think that's a sentence as is. :) Just trying to minimize grok effort for readers.